src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -623,38 +623,33 @@ export class ServerView extends DirMixin(PolymerElement) {
         serverName: String,
         serverHostname: String,
         serverVersion: String,
-        isHostnameEditable: {type: Boolean},
+        isHostnameEditable: Boolean,
         serverManagementApiUrl: String,
         serverPortForNewAccessKeys: Number,
-        isAccessKeyPortEditable: {type: Boolean},
-        serverCreationDate: {type: Date},
+        isAccessKeyPortEditable: Boolean,
+        serverCreationDate: Date,
         serverLocation: String,
         defaultDataLimitBytes: Number,
-        isDefaultDataLimitEnabled: {type: Boolean},
-        supportsDefaultDataLimit: {type: Boolean},
-        showFeatureMetricsDisclaimer: {type: Boolean},
+        isDefaultDataLimitEnabled: Boolean,
+        supportsDefaultDataLimit: Boolean,
+        showFeatureMetricsDisclaimer: Boolean,
         isServerManaged: Boolean,
         isServerReachable: Boolean,
         retryDisplayingServer: Function,
         myConnection: Object,
         totalInboundBytes: Number,
         baselineDataTransfer: Number,
-        accessKeyRows: {type: Array},
+        accessKeyRows: Array,
         hasNonAdminAccessKeys: Boolean,
         metricsEnabled: Boolean,
-        monthlyOutboundTransferBytes: {type: Number},
-        monthlyCost: {type: Number},
-        managedServerUtilzationPercentage: {
```

Comment:
@JonathanDCohen FYI, we don't need to define computed properties, unless an external party needs the value.